client/settings/fraud-protection/components/protection-levels/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useCurrentProtectionLevel, useCurrencies } from 'wcpay/data';
+import {
+	FraudProtectionHelpText,
+	HighFraudProtectionModal,
+	StandardFraudProtectionModal,
+} from '../index';
+import interpolateComponents from 'interpolate-components';
+
+const ProtectionLevels = () => {
+	const [ isStandardModalOpen, setStandardModalOpen ] = useState( false );
+	const [ isHighModalOpen, setHighModalOpen ] = useState( false );
+	const { currencies } = useCurrencies();
+	const storeCurrency = currencies.default ? currencies.default : {};
+
+	const [
+		currentProtectionLevel,
+		updateProtectionLevel,
+	] = useCurrentProtectionLevel();
+
+	const handleLevelChange = ( level ) => {
+		updateProtectionLevel( level );
+	};
+
+	return (
+		<fieldset>
+			<ul>
+				<li>
+					<div className="fraud-protection-single-radio-wrapper">
+						<input
+							name={ 'fraud-protection-level-select' }
+							id={ 'fraud-protection__standard-level' }
+							value={ 'standard' }
+							type={ 'radio' }
+							className={
+								'fraud-protection-single-radio-wrapper__item'
+							}
+							checked={ 'standard' === currentProtectionLevel }
+							onChange={ () => handleLevelChange( 'standard' ) }
+						/>
+						<label htmlFor="fraud-protection__standard-level">
+							<p className="fraud-protection-single-radio-wrapper__item">
```

Comment:
What do you think about merging `<label>` and `<p>`? I think it would make sense after the latest structure changes as the `<p>` is the only direct child of `<label>`.

```js
<label className="fraud-protection-single-radio-wrapper__item" htmlFor="fraud-protection__standard-level">
```

Reply:
Great suggestion. Thanks for catching this. Fixed in [e1a2864](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/e1a2864f2970322d6c65d2d562b92b93addfd395)